web/api/v1/api.go

```diff
@@ -28,9 +28,11 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/prometheus/client_golang/prometheus"
+
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
-	jsoniter "github.com/json-iterator/go"
+	"github.com/json-iterator/go"
```

Comment:
This change isn't needed, I suspect that the alias is there to make obvious that `github.com/json-iterator/go` == `jsoniter`